hub/core/chunk_engine/write.py

```diff
@@ -14,11 +15,19 @@
 from hub.util.array import normalize_and_batchify_shape
 
 
+def write_tensor_meta(key: str, storage: StorageProvider, meta: dict):
+    storage[get_meta_key(key)] = pickle.dumps(meta)
+
+
+def write_dataset_meta(storage: StorageProvider, meta: dict):
+    storage[constants.META_FILENAME] = pickle.dumps(meta)
+
+
 def write_array(
     array: np.ndarray,
     key: str,
-    chunk_size: int,
     storage: StorageProvider,
+    chunk_size: int = 16_000_000,
```

Comment:
I changed the order to give `chunk_size` a default value and now all the previous tests are failing :P

Reply:
weird, the tests pass on my machine

Reply:
ohh, all tests pass it's the benchmarks that fail

Reply:
@McCrearyD That's because I fixed them all

Reply:
Also re-implemented `normalize_and_batchify_array`. Please review.

Reply:
oh ok cool will do

Reply:
im not sure why the benchmarks fail though, on my laptop it takes like 2.5 mins to run them but on circleci it is 10+ m

Reply:
I seem to recall saying something about CI running on 1/4 of a CPU... If many concurrent jobs are running, maybe it gets throttled? Or maybe the matrix is running more benchmarks than we thought